src/TensorFlowNET.Core/Operations/array_ops.cs

```diff
@@ -678,7 +678,6 @@ public static Tensor stop_gradient(Tensor input, string name = null)
             var tape = tf.GradientTape().stop_recording();
             var result = gen_array_ops.stop_gradient(input, name);
             tape.StartRecord();
-            tf.GradientTape().PushTape(tape);
```

Comment:
this removal looks highly suspicious. Does anyone even know why this statement is here?

Reply:
Everything looks fine when I run though all the [examples](https://github.com/SciSharp/SciSharp-Stack-Examples).

Reply:
none of the examples test this specific method though

Reply:
Not deleting this line of code will result in `NULL` gradient when using the `stop_gradient` API.

Reply:
[LayerNorm](https://github.com/SciSharp/TensorFlow.NET/blob/master/src/TensorFlowNET.Keras/Layers/Normalization/LayerNormalization.cs) uses `tf.nn.moment` and `tf.nn.moment` uses the `stop_gradient` API. If this line of code is not deleted, an error will be reported. Deleting this line of code will enable normal training.